apps/api/src/controllers/v1/deep-research.ts

```
@@ -0,0 +1,100 @@
+  success: boolean;
+  id: string;
+};
+
+/**
+ * Initiates a deep research job based on the provided topic.
+ * @param req - The request object containing authentication and research parameters.
+ * @param res - The response object to send the research job ID.
+ * @returns A promise that resolves when the research job is queued.
+ */
+export async function deepResearchController(
+  req: RequestWithAuth<{}, DeepResearchResponse, DeepResearchRequest>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)